pkg/validation/node_conditions.go

```diff
@@ -39,11 +39,11 @@ func findNodeCondition(node *v1.Node, conditionType v1.NodeConditionType) *v1.No
 	return nil
 }
 
-// isNodeReady returns if a Node is considered ready.
+// IsNodeReady returns if a Node is considered ready.
 // It is considered ready if:
 // 1) its Ready condition is set to true
 // 2) doesn't have NetworkUnavailable condition set to true
-func isNodeReady(node *v1.Node) bool {
+func IsNodeReady(node *v1.Node) bool {
```

Comment:
Note that is can return `true` when the node can't actually be productive. A system-node-critical pod, such as the CNI, might not be ready or a static pod, such as kube-proxy, might be absent.